app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -7,6 +7,14 @@ class QuestionsController < ::Questions::QuestionsController
       # default layout for all state file questions
       layout "state_file/question"
 
+      def ip_for_irs
+        if Rails.env.test?
+          "72.34.67.178"
```

Comment:
for passing the feature spec

Reply:
It seems questions controller is a child class of Application controller which already contains this method.  Can we add a clause for `elsif Rails.env.test?` there?